src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -215,7 +222,7 @@ const ChatInfoScreen = () => {
         <AnonymousIcon
           color={channelInfo?.anon_user_info_color_code}
           emojiCode={channelInfo?.anon_user_info_emoji_code}
-          size={normalize(100)}
+          size={dimen.normalizeDimen(100)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The normalization function `dimen.normalizeDimen` is used instead of `normalize` for styling. This change improves consistency in the codebase by using the same normalization function throughout.
```diff
- size={normalize(100)}
+ size={dimen.normalizeDimen(100)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->